airflow/configuration.py

```diff
@@ -274,6 +279,10 @@ class AirflowConfigException(Exception):
 # default_principal = admin
 # default_secret = admin
 
+# QDS credentials, if intergation with Qubole is enabled
+[qds]
+api_token = ''
```

Comment:
We typically store this kind of information in a connection ORM object that we access through a hook